app/src/main/java/ch/epfl/sweng/SDP/game/drawing/DrawingActivity.java

```diff
@@ -293,32 +126,11 @@ public void toolClickHandler(View view) {
         }
     }
 
-    /**
-     * Method that call onDataChange on the UI thread.
-     * @param dataSnapshot Snapshot of the database (mock snapshot
-     *                     in out case).
-     */
-    @VisibleForTesting
-    public void callOnDataChangeTimer(final DataSnapshot dataSnapshot) {
-        this.runOnUiThread(new Runnable() {
-            @Override
-            public void run() {
-                listenerTimer.onDataChange(dataSnapshot);
-            }
-        });
+    public void exitClick(View view) {
+        Log.d(TAG, "Exiting drawing view");
+        this.finish();
     }
 
```

Comment:
This method should only be available to subclass `DrawingOffline` since we do not have the option to leave in online mode.